specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json

```diff
@@ -0,0 +1,924 @@
+        "operationId": "Spatial_PostPointInPolygon",
+        "x-ms-examples": {
+          "PostPointInPolygon": {
+            "$ref": "./examples/Spatial_PostPointInPolygon.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/PositionLatitudeAbbreviated"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/PositionLongitudeAbbreviated"
+          },
+          {
+            "$ref": "#/parameters/PointInPolygonRequestBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PointInPolygonResult"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      },
+      "get": {
+        "description": "**Applies to**: see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThis API returns a boolean value indicating whether a point is inside a set of polygons. The set of polygons is provided by a GeoJSON file which is uploaded via [Data Upload API](https://docs.microsoft.com/rest/api/maps/data/uploadPreview) and referenced by a unique udid. The GeoJSON file may contain Polygon and MultiPolygon geometries, other geometries will be ignored if provided. If the point is inside or on the boundary of one of these polygons, the value returned is true. In all other cases, the value returned is false. When the point is inside multiple polygons, the result will give intersecting geometries section to show all valid geometries(referenced by geometryId) in user data. The maximum number of vertices accepted to form a Polygon is 10,000.\n\n \nTo test this API, you can upload the sample data from [Post Point In Polygon API](https://docs.microsoft.com/rest/api/maps/spatial/postpointinpolygon#examples) examples(Request Body) via [Data Upload API](https://docs.microsoft.com/rest/api/maps/data/uploadPreview) and replace the [udid] from the [sample request below](https://docs.microsoft.com/rest/api/maps/spatial/getpointinpolygon#examples) with the udid returned by Data Upload API.",
+        "operationId": "Spatial_EvaluatePointInPolygon",
```

Comment:
Other APIs use `Get` and `Post` in the title. And just this one uses `Evaluate`. How about changing it to `GetPointInPolygon`.
spectral validation: OperationId for get method should contain "Get" or "list"